### PR TITLE
SREP-1268 Add stub for dedicated-admin-operator

### PR DESCRIPTION
**Pull request by rogbas**
Add configuration for dedicated-admin-operator.

Template path has been updated to point to the new template created by this PR: https://github.com/openshift/dedicated-admin-operator/pull/37

Note: hash is set to none since it's the initial config

### CODE REVIEW

File: dedicated-admin-operator-services/dedicated-admin-operator-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: dedicated-admin-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
Comment (jmelis) [inline dedicated-admin-operator-services/dedicated-admin-operator-operator.yaml]: this file doesn't exist

Comment (rogbas) [reply dedicated-admin-operator-services/dedicated-admin-operator-operator.yaml]: Updated the path reference to: `/build/templates/olm-artifacts-template.yaml.tmpl`
This template is added by the following PR: https://github.com/openshift/dedicated-admin-operator/pull/37

**Comment by jharrington22** [comment]
@jmelis please review

**Review by jmelis — APPROVED** [review]
LGTM

**Pull request merged**